### DISCUSSION TITLE
[New Feature] Where to store avatar images

### DISCUSSION THREAD

**Discussion by piekny27** [piekny27/PulseBook#18] [Q&A] [answered]
How's it going folks? I need a help in chooseing way to store avatar images. Pulsebook is standing on heroku, so i have to store at external services. Best comment wins pinned post.

**Comment by OlexTix** [comment, marked as answer]
A good idea would be either to use a 2nd server for storage or use it to create a WebAPI (such as REST API). If you choose the latter, you will have to send the images in Base64EncodedString, then convert them back to bitmap and save them. Next, you must specify their filepaths in the 1st server's database, so the server can fetch images.

**Reply by piekny27** [reply]
Thanks!
